src/containers/Network/Nodes.tsx

```diff
@@ -44,7 +44,7 @@ export const Nodes = () => {
         const nodes: NodeData[] = allNodes.map((node: NodeResponse) => ({
           ...node,
           version: node.version?.startsWith('rippled')
```

Comment:
Should this use handleNodeVersion too?

Reply:
The node table will add a * toward the versions with rippled hash so it is slightly different from the chart